<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-07-07T15:13:43 IST" name="Suitecheck" finished-at="2022-07-07T15:14:12 IST" duration-ms="29308">
    <groups>
    </groups>
    <test started-at="2022-07-07T15:13:43 IST" name="Testcheck" finished-at="2022-07-07T15:14:12 IST" duration-ms="29308">
      <class name="com.sn.Testcases.ChildIncidentTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.sn.Testcases.ChildIncidentTest@59a008ba]" started-at="2022-07-07T15:13:44 IST" name="setup" finished-at="2022-07-07T15:14:04 IST" duration-ms="20268" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="createchildincident(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.sn.Testcases.ChildIncidentTest@59a008ba]" started-at="2022-07-07T15:14:04 IST" name="createchildincident" data-provider="getchildincidentdata" finished-at="2022-07-07T15:14:11 IST" duration-ms="6943" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Abel Tuter]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[expected:<false> but was:<true>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: expected:<false> but was:<true>
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.failNotEquals(Assert.java:282)
at junit.framework.Assert.assertEquals(Assert.java:64)
at junit.framework.Assert.assertEquals(Assert.java:149)
at junit.framework.Assert.assertEquals(Assert.java:155)
at com.sn.Pages.IncidentCreate.CreateNewIncident(IncidentCreate.java:104)
at com.sn.Testcases.ChildIncidentTest.createchildincident(ChildIncidentTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createchildincident -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.sn.Testcases.ChildIncidentTest@59a008ba]" started-at="2022-07-07T15:14:11 IST" name="teardown" finished-at="2022-07-07T15:14:12 IST" duration-ms="730" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.sn.Testcases.ChildIncidentTest -->
    </test> <!-- Testcheck -->
  </suite> <!-- Suitecheck -->
</testng-results>
